java/src/main/java/com/saucelabs/saucebindings/SauceOptions.java

```diff
@@ -129,6 +130,22 @@
         knownCITools.put("TeamCity", "TEAMCITY_PROJECT_NAME");
     }
 
+    /**
+     * @return SauceConfigsChrome,
+     *   note: constructor implementation is deprecated for public use, but usage here is intentional
```

Comment:
Oh, right, I see what you are saying. We're going to deprecate it for the next release, then change it to protected/private for next major release. Wanted to give a warning about it first.

Reply:
Oh, right, this is a new method. got it. (trying to remember what I was doing 2 months ago is hard)

Reply:
and... it's because I have it in a separate package (`com.saucelabs.saucebindings.configs` vs `com.saucelabs.saucebindings`. Any advice on a better way to organize this? I feel like everything should probably be in an options package, but I also think it is considered bad to change the package from one version to another? :) Would it make sense to have everything in the main package?